frontend/src/services/fetchData.ts

```diff
@@ -83,11 +90,40 @@ export const fetchApplicationAdmins = async (): Promise<FamAppAdminGetResponse[]
  * @param applicationAdminId id to delete fam_user_role_assignment record.
  */
 export const deleteAndRefreshApplicationAdmin = async (
-    applicationAdminId: number,
+    applicationAdminId: number
 ): Promise<FamAppAdminGetResponse[]> => {
     await AdminMgmtApiService.applicationAdminApi.deleteApplicationAdmin(
         applicationAdminId
     );
     // When deletion is successful, refresh (fetch) for frontend state.
     return fetchApplicationAdmins();
 };
+
+export const fetchDelegatedAdmins = async (
+    applicationId: number | undefined
+): Promise<FamAccessControlPrivilegeGetResponse[]> => {
+    if (!applicationId) {
```

Comment:
so let's keep the check inside here, and add
` if (!applicationId && !FamLoginUserState.isAdminOfSelectedApplication()) { return [] ;}`

and no need the if condition below